patches/react-native-progress-bar-animated+1.0.6.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/node_modules/react-native-progress-bar-animated/src/AnimatedProgressBar.js b/node_modules/react-native-progress-bar-animated/src/AnimatedProgressBar.js
+index 29a0f82..b5e6774 100644
+--- a/node_modules/react-native-progress-bar-animated/src/AnimatedProgressBar.js
++++ b/node_modules/react-native-progress-bar-animated/src/AnimatedProgressBar.js
+@@ -26,8 +26,8 @@ class ProgressBar extends React.Component {
+     }
+   }
+ 
+-  componentWillReceiveProps(props) {
+-    if (props.value !== this.state.progress) {
++  static getDerivedStateFromProps (props,state) {
```

Comment:
better, but 2 things. one of which is optional.
in 5 years, we will not remember why we are patching this, so add a comment on top of the new code explaining what error/warning are you fixing. so when the packages get updated and we wanna remove the patch, we know if it's safe to delete the patch. and be specific on how to repro the warning you're fixing.
so like `removing btingan warning that appears when the progress bar is rendered`

2. look into the package github issues and see if someone has mentioned. if so, you'll either find people suggesting fixes, or someone sent a fix PR. for either cases, add a comment here with the link that has the fix, basically referencing where did you get this fix from.

Reply:
comment added